roxie/ccd/ccdcontext.cpp

```diff
@@ -2786,7 +2786,7 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
 
     void initDebugMode(bool breakAtStart, const char *debugUID)
     {
-        if (!debugPermitted || !ownEP.port || !nativeProtocol)
+        if (!debugPermitted || !ownEP.port || nativeProtocol)
```

Comment:
Trivial, but error message isn't quite right for the last check, nativeProtocol, which just indicates something about the request, not how this system is configured.

Reply:
True, though not new.